examples/vasp/wallet.py

```diff
@@ -107,11 +107,14 @@ def pay(
         """make payment from given user account to intent_id"""
 
         intent = identifier.decode_intent(intent_id, self.hrp)
+        if not intent.amount or not intent.currency_code:
+            raise ValueError("should provide currency_code and amount")
```

Comment:
why raise error if one or both of them are none?

Reply:
to keep this example simple. 

the example only works for offchain payment, no need to accept all kinds of input.